src/bindings/js/node/CMakeLists.txt

```diff
@@ -69,7 +75,12 @@ target_include_directories(${PROJECT_NAME} PRIVATE
     "${CMAKE_CURRENT_SOURCE_DIR}/include"
 )
 
-target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime)
+target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime ${CMAKE_JS_LIB})
+
+if(MSVC AND CMAKE_JS_NODELIB_DEF AND CMAKE_JS_NODELIB_TARGET)
+  # Generate node.lib
+  execute_process(COMMAND ${CMAKE_AR} /def:${CMAKE_JS_NODELIB_DEF} /out:${CMAKE_JS_NODELIB_TARGET} ${CMAKE_STATIC_LINKER_FLAGS})
```

Comment:
We need this file to avoid VS compilation error (like this: `error LNK2019: unresolved external symbol napi_get_last_error_info`). For other platforms we use flag `no-missing-declarations` to avoid same issue.
Official way is to use **CMake.js** npm package to perform node-addon compilation (https://nodejs.org/api/n-api.html#cmakejs). I took part, that you are asking, from this recommended config: https://www.npmjs.com/package/cmake-js#node-api-and-node-addon-api

Because we are using native **CMake** to compilation, we cannot perform npm script running to compile js bindings.
Without this package we have to prepare [node-lib.def](https://github.com/openvinotoolkit/openvino/pull/21995/files#diff-2ee8ca9dba41c70fa7b0cf7bfe5eeab957331f27070137b5cca22e58557302e2) file by own. 
**CMake.js** prepare file node-lib.def, based on symbols from specific node-api-headers version. This file creates and puts in build directory by this script: https://github.com/cmake-js/cmake-js/blob/v7.2.1/lib/cMake.js#L293
It takes list of symbols from specific version from another package and write this file into build dir.

I repeat the same logic, but using **CMake** only.